### DISCUSSION TITLE
Sort feature back

### DISCUSSION THREAD

**Discussion by aafritz1** [PaulWoitaschek/Voice#1907] [Ideas]
It would be great to have the sort feature back in the library of 'currently reading' and 'not started' books. I just found the search feature which is great, thank you for that. This is an awesome app, great job!

**Comment by MNeMoNiCuZ** [comment]
Yes! This please.
The new layout is prettier, but it would be great to be able to sort by "Newest added" "Author" " Book title", similar to how it was like before.
